appendix_meta_rules.md

```diff
@@ -570,8 +584,9 @@ A brief title for the rule that should contain what the rule is supposed to dete
 
 must be `filter`
 
-### Change to Condition 
+### Change to Condition
 
+<!-- What would be an include filter? Something added as "or" to the referenced rules instead of "and not"? -->
```

Comment:
`action` can be remove too now.
For me can be only "and" or "and not",  "or" means to change the detection logic.
There have been discussions on it, for me 99.99% of filters are "and not".
The idea is to have the possibility of awakening the "and not not" = "and" (or not 😆 ).

Why not a "If omited the value is `exclude`" ?

Reply:
It doesn't really answer my question what an include filter would be or what it would be used for. I cannot imagine it right now. If you want to include more events to be able to match the original rule, it could only be achieved by adding the include filter with an "or" logic to the original condition.

If we say we just use "and" instead of "and not" to add to the condition, it makes more sense in my opinion, but I wouldn't name it "include filter" then. Maybe this is a feature which we just don't want to support. It makes it harder to understand and to explain and I don't really see the use case. As you said, I also see 99.99% of cases with "and not". Adding something to the filter using "and" would mean a change in the detection logic which means it probably should be a new rule.

Reply:
Why not simplify "global filter" is an "and not" ?
So remove the include or exclude.

Reply:
Yes, that's what I suggested somewhere in that big blob of text of mine xD. So I agree :)

Reply:
An include can be a computer; a date, a user or practically any metadata like field that can be used to group something to avoid FPs. Then again it'll be confusing as this is called a filter and inclusion in filters doesn't really sound that great.